package.json

```diff
@@ -25,7 +25,8 @@
   "dependencies": {
     "@actions/core": "^1.4.0",
     "@actions/exec": "^1.1.0",
-    "@actions/github": "^5.0.0"
+    "@actions/github": "^5.0.0",
+    "request": "^2.88.2"
```

Comment:
The request package is currently deprecated. Look at using [node-fetch](https://github.com/node-fetch/node-fetch)—it's based on the WHATWG Fetch API available to modern browsers so it has a very mature API and is backed by a lot of documentation. Additionally, there are TypeScript type definitions available from the DefinitelyTyped project in the form of the [\@types/node-fetch](https://www.npmjs.com/package/@types/node-fetch) package.

Reply:
:+1: I think @drazisil looked into using vanilla JS for this, if not, I'll pull down `node-fetch`